### PR TITLE
chore(scripts-puppeteer): try closing open about:blank to free up memory

### PR DESCRIPTION
**Pull request by Hotell**
## Behavior

Puppeteer page open timeout errors still continue. Added helper to try another thing that might help to improve the situation.

## Related Issue(s)

* Follows https://github.com/microsoft/fluentui/pull/26375

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8e9b5bf66f027b40ef7ccf5272940a343d330446'>8e9b5bf66f027b40ef7ccf5272940a343d330446</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

File: scripts/puppeteer/src/utils.ts [hunk 1]
```diff
@@ -30,6 +30,32 @@ export async function launch(options: LaunchOptions = {}) {
   return browser;
 }
 
+/**
+ *
+ * Try out https://github.com/puppeteer/puppeteer/issues/1490
```
Comment (Hotell) [inline scripts/puppeteer/src/utils.ts:35]: ok this won't fix anything , pipeline failed 
image

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Comment by Hotell** [comment]
https://github.com/microsoft/fluentui/pull/26461#discussion_r1084389059

**Pull request closed without merging**